examples/case_studies/bart_heteroscedasticity.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>nitpick, instead of computed y_hdi, you can just do </p><p> </p><p>pps = az.extract(posterior_predictive_marketing_full, group="posterior_predictive", var_names=["y"]).T</p><p> </p><p>and then </p><p> </p><p>az.plot_hdi(</p><p>&nbsp;&nbsp;&nbsp;x=X[:, 0], y=pps, ax=ax, fill_kwargs={"alpha": 0.3, "label": r"Likelihood $94\%$ HDI"}</p><p>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/519/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>cool! fixed in https://github.com/pymc-devs/pymc-examples/pull/519/commits/a5b2acdf0ef1462da61531e575569adc045db660</p>